src/models/Event.js

```diff
@@ -11,7 +11,6 @@ const USER_TYPE={
 const _schema = new Schema({
   title: { type: String, default: '' },
   scheduled_date: { type: String, default: '' },
-  duration: { type: String, default: '' },
   status: { type: String, default: '' },
   duration: { type: Number, default: 0 },
```

Comment:
add the enum check with
{ type: String, enum: Object.values(enums.FILE_MODULES), default: enums.FILE_MODULES.DEFAULT },